### PR TITLE
Admin list

### PR DESCRIPTION
**Pull request by MarwaKhalid**
Issue #238

### CODE REVIEW

File: Admin-app/src/backend/firebaseAdminUtilities.js [hunk 1]
```diff
@@ -49,10 +49,46 @@ const getAdminRef = (key) => {
   return getDocRef(getTableName(), key);
 };
 
+const setDisabled = async (patientKey) => {
+  console.log("[setStatus]" + patientKey);
```
Comment (smarkandu) [inline Admin-app/src/backend/firebaseAdminUtilities.js:53]: This should be taken out (copy/paste probably).

File: Admin-app/src/components/AdminList/index.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
+/**
```
Comment (smarkandu) [inline Admin-app/src/components/AdminList/index.js:1]: Maybe for disabled accounts, they should be shown as greyed out (or less bright).

File: Admin-app/src/components/SignIn/index.js [hunk 3]
```diff
@@ -75,38 +75,43 @@ export default function SignIn() {
   const [email, setEmail] = useState("");
```
Comment (smarkandu) [inline Admin-app/src/components/SignIn/index.js:75]: Why are we touching the "SignIn" page for this feature?  Put in by accident?

Seems to be a fix on the bugs @tusharraval102 is working on.  You should probably revert this.

**Review by smarkandu — CHANGES_REQUESTED** [review]
Changes to the "Sign In" seem to be outside the scope of this feature.  Since @tusharraval102 already has PRs for this, this part probably should be reverted.

File: Admin-app/src/backend/firebaseAdminUtilities.js (hunk 1 above)
Comment (MarwaKhalid) [reply Admin-app/src/backend/firebaseAdminUtilities.js:53]: Yes, I didn't notice thank you

File: Admin-app/src/components/SignIn/index.js (hunk 3 above)
Comment (MarwaKhalid) [reply Admin-app/src/components/SignIn/index.js]: I fixed this to the way Tushar was approaching, but I'll check out his branch and discuss further with you on the server

File: Admin-app/src/components/AdminList/index.js (hunk 2 above)
Comment (MarwaKhalid) [reply Admin-app/src/components/AdminList/index.js:1]: Done

**Review by smarkandu — CHANGES_REQUESTED** [review]
The SignIn page changes need to be clarified.  There's no reason for this file to be changed in the context of this bug.  Maybe formatting issues again?

**Review by smarkandu — DISMISSED** [review]
Okay it looks like you have VS Code formatting issues.  But I was able to save to your branch using my own settings to reduce the changes shown for the Sign In page.

However I noticed that an administrator is able to disable themselves.  We need to discuss if we should permit that functionality, as normally it's not really something that a person would want to do.

**Review by SamanthaGuillemette — APPROVED** [review]
All good!

**Review by SamanthaGuillemette — APPROVED** [review]
Good!

**Review by SamanthaGuillemette — APPROVED** [review]
good!

**Review by SamanthaGuillemette — APPROVED** [review]
good

**Review by SamanthaGuillemette — APPROVED** [review]
good

**Pull request merged**